### PR TITLE
Wallet: Fix incorrect constructor JavaDoc

### PR DESCRIPTION
**Pull request by msgilligan**
The (canonical) constructor does not create a wallet with a random seed.

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/wallet/Wallet.java [hunk 1]
```diff
@@ -620,8 +620,8 @@ else if (header == params.getBip32HeaderP2WPKHpub() || header == params.getBip32
     }
 
     /**
-     * Creates a new, empty wallet with a randomly chosen seed and no transactions. Make sure to provide for sufficient
-     * backup! Any keys will be derived from the seed. If you want to restore a wallet from disk instead, see
+     * Creates a new, empty wallet with no transactions. Make sure to provide for sufficient backup!
```
Comment (schildbach) [inline core/src/main/java/org/bitcoinj/wallet/Wallet.java:623]: Hmm, good catch.

File: core/src/main/java/org/bitcoinj/wallet/Wallet.java [hunk 2]
```diff
@@ -620,8 +620,8 @@ else if (header == params.getBip32HeaderP2WPKHpub() || header == params.getBip32
...
+     * Any keys will be derived from the seed. If you want to restore a wallet from disk instead, see
```
Comment (schildbach) [inline core/src/main/java/org/bitcoinj/wallet/Wallet.java:624]: But shouldn't this line go away as well? (the hint about restoring from disk maybe reworded)

File: core/src/main/java/org/bitcoinj/wallet/Wallet.java (hunk 1 above)
Comment (msgilligan) [reply core/src/main/java/org/bitcoinj/wallet/Wallet.java:623]: I always check the JavaDoc when people are asking question or having problems with the API. This was driven by a conversation in our Matrix group.

File: core/src/main/java/org/bitcoinj/wallet/Wallet.java (hunk 2 above)
Comment (msgilligan) [reply core/src/main/java/org/bitcoinj/wallet/Wallet.java:624]: Yes, give me a few minutes and I'll force-push an update.

**Comment by msgilligan** [comment]
@schildbach rebased, updated, and force-pushed.

File: core/src/main/java/org/bitcoinj/wallet/Wallet.java (hunk 2 above)
Comment (msgilligan) [reply core/src/main/java/org/bitcoinj/wallet/Wallet.java]: Done.

**Comment by schildbach** [comment]
Merged.
